src/platform/backends/shared/base_snapshot.h

```diff
@@ -21,6 +21,7 @@
 #include <multipass/snapshot.h>
 
 #include <multipass/memory_size.h>
+#include <multipass/virtual_machine_description.h>
```

Comment:
Small detail, but you should be able to fwd declare `VirtualMachineDescription` here and include only in the `cpp`.

Reply:
Right, done.